ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml

```diff
@@ -55,28 +59,24 @@ tests:
       ENABLE_AUTOSCALING: "false"
       MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.12"
+      PODS_PER_NODE: "100"
+      READY_WAIT_TIMEOUT: 30m
       REPLICAS: "24"
     test:
-    - ref: openshift-qe-cluster-density-v2
+    - chain: openshift-qe-node-density
     workflow: rosa-aws-sts
-- as: perfscale-rosa-multiaz-node-density-v2-24nodes-periodic
-  cron: 0 8 * * 3
+- always_run: false
+  as: perfscale-rosa-data-path-3nodes
   steps:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
-      COMPUTE_MACHINE_TYPE: m5.xlarge
```

Comment:
should we just keep these variable for future runs (if someone was to turn this back on) would be consistent configurations

Reply:
I actually didn't make any changes to the test.
And not sure what all variables are missing for it, will just delete the test for now as I think it was added very early stages and is a non periodic test